### PR TITLE
Update changelog for 1.3.1

### PR DESCRIPTION
**Pull request by pschorf**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -4,6 +4,11 @@ All notable changes to this project will be documented in this file
  
 The format is based on [Keep a Changelog](http://keepachangelog.com/) and this project adheres to [Semantic Versioning](http://semver.org/).
  
+## [1.3.1] - 2017-05-18
+### Changed
+- Make DRU order deterministic, from @wyegelwel
+- Remove concat usage, from @pschorf
+- Use agent to ensure processing order, from @shams
```
Comment (dposada) [inline CHANGELOG.md]: This one seems too vague. Processing order of what?

Comment (pschorf) [inline CHANGELOG.md]: Removed

File: CHANGELOG.md [hunk 2]
```diff
@@ -4,6 +4,10 @@ All notable changes to this project will be documented in this file
  
 The format is based on [Keep a Changelog](http://keepachangelog.com/) and this project adheres to [Semantic Versioning](http://semver.org/).
  
+## [1.3.1] - 2017-05-18
+### Changed
+- Make DRU order deterministic, from @wyegelwel
```
Comment (wyegelwel) [inline CHANGELOG.md]: The default cycle time for killing tasks that exceed max run time was dropped from 10 minutes to 1 minute as well

Comment (pschorf) [inline CHANGELOG.md]: Added

**Pull request merged**